src/components/PatientLookup.tsx

```diff
@@ -24,15 +21,15 @@ interface PatientLookupProps {
   isOnDashboard: boolean
 }
 
-const PatientLookup = (props: PatientLookupProps) => {
+const PatientLookup: React.FC<PatientLookupProps> = ({ isOnDashboard }) => {
   const { dispatch } = useContext(AppContext);
   const [privacyLevel, setPrivacyLevel] = useState(PrivacyLevel.ANONYMIZE_ALL_DATA)
   const [patientID, setPatientID] = useState<string>("");
   const history = useHistory();
 
   const [isDropDownOpen, setDropDownOpen] = React.useState(false);
 
-  const onSelect = (value: any) => {
+  const onSelect = (_value: any) => {
```

Comment:
@colinywu could you please replace `any` with the appropriate type?

I believe the function parameter is `event?: React.SyntheticEvent<HTMLDivElement>`.

Reply:
Removed the value field, as it was unused and unecessary

Reply:
The common practice is to either not add the unused function parameter to the function signature or use _ before the name of the parameter (ex. `_value: React.SyntheticEvent<HTMLDivElement>`) to indicate that it is unused. The benefit of having _ in front of the function parameter is it lets the contributors be aware of what parameters are available to be used.

Both of approaches are fine in my opinion and as a team, if needed, we could discuss it further to create our own set of "Best Practices" needed for this project.